BaselinePitch: 0.5
BaselineYaw: 15.0
PitchThreshold: 20.0
YawThreshold: 10.0